src/toast/pixels.py

```diff
@@ -921,6 +921,125 @@ def sync_alltoallv(self, local_func=None):
         self.reverse_alltoallv()
         return
 
+    @function_timer
+    def stats(self, comm_bytes=10000000):
+        """Compute some simple statistics of the pixel data.
+
+        The map should already be consistent across all processes with overlapping
```

Comment:
Any way to check if the map was synchronized?

Reply:
I suppose we could have each process compute a hash of their local submap data and compare across all processes that had the same submap.  This would require some extra coding.  Since we are only using this in the benchmarks at the moment where we control what has been done to the map data, I would prefer to punt that to the future if possible.